lib/fastlane/plugin/revenuecat_internal/helper/versioning_helper.rb

```diff
@@ -160,13 +160,18 @@ def self.detect_bump_type(version_name_a, version_name_b)
         end
       end
 
-      private_class_method def self.latest_non_prerelease_version_number
-        Actions
-          .sh("git tag", log: false)
-          .strip
-          .split("\n")
-          .select { |tag| tag.match("^[0-9]+.[0-9]+.[0-9]+$") }
-          .max_by { |tag| Gem::Version.new(tag) }
+      private_class_method def self.latest_version_number(include_prerelease: false)
```

Comment:
I've been feeling as I read this that I like the plural `include_prereleases` more. Thoughts?

Reply:
I... completely agree with that! Will change 😊